src/dialog/tooltip/ellipsis-tooltip.directive.ts

```diff
@@ -35,12 +36,32 @@ export class EllipsisTooltip extends TooltipDirective {
 	 * @returns null
 	 * @memberof EllipsisTooltip
 	 */
-	toggle() {
+	generateTooltipContent() {
```

Comment:
perhaps `updateTooltipContent` might be more accurate?

Reply:
true!